sql-cli/tests/test_connections.py

```diff
@@ -41,3 +41,37 @@ def test_validate_connections_config_file_does_not_contain_connection(
     captured = capsys.readouterr()
     assert f"Config file does not contain given connection {unknown_connection_id}" in captured.out
     assert f"AIRFLOW_CONN_{unknown_connection_id.upper()}" not in os.environ
+
+
+@patch("sql_cli.connections.Path.is_file", return_value=True)
+@patch("sql_cli.connections.Connection.test_connection", return_value=(True, ""))
+@patch.dict(os.environ, {}, clear=True)
+def test_validate_connections_multiple_connections(
+    mock_test_connection,
+    mock_is_file,
+    multiple_connections,
+    initialised_project,
+    capsys,
+):
+    validate_connections(connections=multiple_connections)
+    captured = capsys.readouterr()
+    assert captured.out.count("PASSED") == 2
+    assert "AIRFLOW_CONN_SQLITE_CONN1" in os.environ
+    assert "AIRFLOW_CONN_SQLITE_CONN2" in os.environ
+
+
+@patch("sql_cli.connections.Path.is_file", return_value=True)
+@patch("sql_cli.connections.Connection.test_connection", return_value=(True, ""))
+@patch.dict(os.environ, {}, clear=True)
+def test_validate_connections_multiple_connections_single_specified(
+    mock_test_connection,
+    mock_is_file,
+    multiple_connections,
```

Comment:
hmm why have this as a fixture and not part of this test directly?

I feel like duplication of code is better in this case than to jump to look for definition of that fixture unless we plan to use that at several places

Reply:
A good read about the same is: https://testing.googleblog.com/2019/12/testing-on-toilet-tests-too-dry-make.html

Reply:
@kaxil, I agree. I was naively being consistent with what we already had in this test file (connections fixture) - but I agree with your view. I already changed it, please, have a look